packages/e2e/features/ui/components/authenticator/sign-in-totp-mfa.feature

```diff
@@ -60,19 +61,21 @@ Feature: Sign In with TOTP MFA
 @angular @react @vue
   Scenario: Successful sign up shows correct username from authenticated user
     When I click the "Create Account" tab
+    Given I intercept '{ "headers": { "X-Amz-Target": "AWSCognitoIdentityProviderService.SignUp" } }' with fixture "sign-up-with-email"
     And I type a new "email"
     And I type my password
     And I confirm my password
-    Given I intercept '{ "headers": { "X-Amz-Target": "AWSCognitoIdentityProviderService.SignUp" } }' with fixture "sign-up-with-email"
```

Comment:
Why does this move up?

Reply:
This matches our other tests, that have this line of code right after `confirm my password`. It doesn't really matter where this is, as long as it's before they click the create account button.